andela/ah-frontend-kronos
# #166879998 Implement CRUD on articles

#### What does this PR do?
- Enable to creating my articles and read/access articles created by other authors. 
- Enable update and delete  **ONLY**  articles that I have authored.

#### Description of Task to be completed?

- [x] Create articles using the frontend UI

- [x] Read articles using the frontend UI

- [x] Update articles using the frontend UI

- [x] Delete articles using the frontend UI

#### How should this be manually tested?
- Pull branch `ft-article-CRUD-166879998`
- Start server with `npm start`
- You should be able to see the articles on the landing page.
- To read an article click on the `article image` or `article title`.

#### Any background context you want to provide?
Articles form the backbone of our application and introduce us to CRUD which is the building block of most web applications.
#### What are the relevant pivotal tracker stories?
[#166879998](https://www.pivotaltracker.com/story/show/166879998)

#### Screenshots
Display articles as landing page
<img width="1435" alt="Screenshot 2019-07-20 at 12 11 12" src="https://user-images.githubusercontent.com/9926422/61576885-68dd9f00-aae8-11e9-9861-cd7dbefb3134.png">

Upon successful creation of an article, you are redirected to that article where you can edit or delete an article
<img width="1418" alt="Screenshot 2019-07-20 at 12 13 30" src="https://user-images.githubusercontent.com/9926422/61576893-81e65000-aae8-11e9-9a55-c0e9ccbd8ab6.png">

Upon successful creation update of an article, you are redirected to that article where you can see the changes reflecting
<img width="1436" alt="Screenshot 2019-07-20 at 12 14 29" src="https://user-images.githubusercontent.com/9926422/61576907-b22dee80-aae8-11e9-9260-a3edca887e01.png">

Capability to delete after seeking for consent

<img width="1429" alt="Screenshot 2019-07-20 at 12 14 57" src="https://user-images.githubusercontent.com/9926422/61576916-d1c51700-aae8-11e9-8538-0f935043ada5.png">
